pulsar-io/jdbc/core/src/main/java/org/apache/pulsar/io/jdbc/JdbcAbstractSink.java

```diff
@@ -91,7 +92,11 @@ public void open(Map<String, Object> config, SinkContext sinkContext) throws Exc
 
         Class.forName(JdbcUtils.getDriverClassName(jdbcSinkConfig.getJdbcUrl()));
         connection = DriverManager.getConnection(jdbcSinkConfig.getJdbcUrl(), properties);
-        connection.setAutoCommit(false);
+        try {
+            connection.setAutoCommit(false);
```

Comment:
if you cannot set "autoCommit" then you must not call commit or rollback

Reply:
Thanks, I was negligent.